ci/ci_steps.sh

```diff
@@ -1,5 +1,5 @@
 #!/bin/bash
-ENVOY_BUILD_SHA=f7f2a34fe7dcfb39a81ed21728c8df2af223bbb0
+ENVOY_BUILD_SHA=22c55f8ec756c5ddeb26c3424e128a91aec23116
```

Comment:
Because we only rebuild the build image automatically when the files in the build image creation directory change. This is the last time they did.